.changeset/quiet-pianos-end.md

```diff
@@ -0,0 +1,11 @@
+---
+'@backstage/plugin-scaffolder-backend-module-bitbucket-cloud': patch
```

Comment:
Think we can probably split this up into multiple changesets so that the description matches the packages affected rather than having multiple changes in one changeset.